project/Dependencies.scala

```diff
@@ -340,8 +340,7 @@ object Dependencies {
 
   val MongoDb = Seq(
     libraryDependencies ++= Seq(
-        "org.mongodb" % "mongodb-driver-reactivestreams" % "1.12.0", // ApacheV2
-        "org.mongodb.scala" %% "mongo-scala-bson" % "2.7.0" % "test" // ApacheV2
+        "org.mongodb.scala" %% "mongo-scala-driver" % "4.1.0" // ApacheV2
```

Comment:
We are ready for Alpakka 3.0 merges now!
There is a later release of the Mongo driver (4.1.1).
What's the value-add of `mongo-scala-driver` compared to `mongodb-driver-reactivestreams` and `mongo-scala-bson` as used before?

Reply:
@ennru AFAIK, the main 4.x Java driver lib now has all APIs in the same distribution package. The separate Scala driver and reactivestreams driver have been discontinued during the release of their 4.x driver.

The main driver page describes that here: https://mongodb.github.io/mongo-java-driver/

It's been a while again now that I looked at the detailed differences but judging from my own comments above the migration process is mostly one of changing the imported packages.

Reply:
The old driver has been superseded by the new one. According to the [driver compatibility table](https://docs.mongodb.com/drivers/scala/#mongodb-compatibility), only the new driver is compatible with MongoDB 4.4+.

Reply:
Thank you for this references. Please add them in a comment for later reference.
Please update even the docs page in `mongodb.md` with appropriate links to give information about the driver and compatible MongoDB versions.

Reply:
-  Not sure if by _add them in a comment_ you mean here on this PR or somewhere else. 
-  `mongodb.md` updated with information about the new driver.
Regarding this doc, I'd remove the _Alternative connector_ section as it is not [compatible](http://reactivemongo.org/releases/1.0/documentation/release-details.html#compatibility) with MongoDB version 4.4, at least for now. I haven't removed it just in case you want to keep it for other reasons. 
Thanks!